model/github_runner.rb

```diff
@@ -16,15 +16,18 @@ class GithubRunner < Sequel::Model
 
   dataset_module do
     def total_active_runner_vcpus
+      predefined_runner_labels = Github.predefined_runner_labels
+      custom_label_aliases = GithubCustomLabel.all.to_h { [it.label, it.alias_for] }
```

Comment:
Since `GithubCustomLabel.label` is unique per installation, should we only fetch the labels that belong to the runner’s installation?